typo3Model.xml

```diff
@@ -41,6 +41,27 @@
       <type name="dkd:typo3:pages">
           <title>TYPO3 Page</title>
           <parent>cm:folder</parent>
+          <properties>
+              <property name="dkd:typo3:pages:doktype">
```

Comment:
That is indeed a fair point, but I opted to not do that (currently) for two reasons: 1) TYPO3 allows custom page doktypes to be declared and used and it's not feasible imho to require all those as custom CMIS sub-types, and 2) if at some point we do opt to declare CMIS types for each doktype, we would have to implement yet another mapping based on for example TypoScript.

I know this isn't the perfect solution but this strategy allows me to avoid over-engineering the model too early in the process ;)